tensorflow/c/kernels.cc

```diff
@@ -81,6 +81,9 @@ void AddTypeConstraint(TF_KernelBuilder* kernel_builder, const char* attr_name,
   // TF_CALL_ALL_TYPES macro can find tensorflow::string as string.
   switch (dtype) {
     TF_CALL_ALL_TYPES(CASE);
+    TF_CALL_QUANTIZED_TYPES(CASE);
+    TF_CALL_quint16(CASE);
```

Comment:
@jianlijianli do you know why we do not include quint16 and qint16 in `TF_CALL_QUANTIZED_TYPES`?

Reply:
That macro was added quite a while ago and I don't have much context but it seems quint16 and qint16 was not needed in the original use case but there is a [TODO](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/register_types.h#L205-L207) to include those types.

Reply:
Ok, thanks for clarifying. I guess this could be cleaned up separately.